ksqldb-api-client/src/main/java/io/confluent/ksql/api/client/StreamInfo.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2020 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.api.client;
+
+/**
+ * Metadata for a ksqlDB stream.
+ */
+public interface StreamInfo {
+
+  /**
+   * Returns the name of this stream.
+   *
+   * @return stream name
```

Comment:
nit, my personal preference is just to have `@return the name of this stream` as one line if it's a simple getter